internal/cmd/options.go

```diff
@@ -43,6 +45,15 @@ func (opts *LoggingFlagOpts) ApplyFlags(fs *pflag.FlagSet) {
 
 // SetLoggerLevel sets up the logger based on common options.
 func (opts *LoggingFlagOpts) SetLoggerLevel(ctx context.Context) context.Context {
+	logger := log.GetLogger(ctx)
```

Comment:
We should call `log.GetLogger()` after `trace.WithLoggerLevel()`

Reply:
changed accordingly, refactored this function.